assets/js/base/context/store-notices-context.js

```diff
@@ -52,6 +56,16 @@ export const StoreNoticesProvider = ( {
 		[ createNotice, context ]
 	);
 
+	const createSnackbarNotice = useCallback(
```

Comment:
I'm wondering about whether it's nicer to expose snackbars using the same API as the existing notices - i.e. do we need `createSnackbarNotice` as a convenience?

Reply:
See my comment [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1973#discussion_r392724837)